pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -871,6 +871,7 @@ type ClusterNetworkingConfig struct {
 	CoreDNSReplicas *int32 `json:"coreDNSReplicas,omitempty"`
 
 	// KonnectivityEnabled enables konnectivity for controlplane to node network communication.
+	//Deprecated: As OpenVPN will be removed in the future, clusters with konnectivity disabled will not be supported.
```

Comment:
Nit: The `Deprecated:` should be prefixed to the first comment line of the field. In addition, please clarify the "not be supported" part a little bit more so users understand _when_ it will stop being supported. It should. say at least "... will not be supported in future KKP versions.", IMHO.

Reply:
Probably we should also mention here to use this field to migrate older clusters with OpenVPN to the Konnectivity

Reply:
Good point. I see you added it to the comment already, thanks!